.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+name: CI - Build only (deploy if master)
+
+on: 
+  push:
+    branches:
+    - '**'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Makes sense. 20.04 seems reasonable.

Reply:
Actually I believe ubuntu-latest is the latest LTS version (in this case 20.04). I will confirm.

Reply:
https://github.blog/changelog/2020-10-29-github-actions-ubuntu-latest-workflows-will-use-ubuntu-20-04/

https://github.com/actions/virtual-environments/issues/1816

Reply:
Yes, `latest` is the latest LTS version, 20.04 right now. This will stay supported at least until the end of 2024, so it should be fine to use this explicitly. If we use `latest` then the stack will change without our input at some point in 2022, potentially resulting in undesirable behaviour.

Reply:
there are lots of announcements prior to them cutting over... so it is probably a pretty safe bet. Also given this is a node execution I doubt the impact will be high. I mean at the end of the day it probably matters little either way.

Reply:
Moved to 20.04 explicitly.